### DISCUSSION TITLE
UVTools acceleration gcode gives 13h03 instead of 6h38 (or even 03h31) !?

### DISCUSSION THREAD

**Discussion by Alomphega** [sn4k3/UVtools#486] [Q&A]
Hello,

I just saw my last print duration was 13h03, while the previous one was 06h38 on the same printer with the almost same pieces !
Chitubox gave 03h07'06'' and UVTools 03h31' for this 13h03' print...

I used your "uvtools_acceleration.gcode", and also the "Stir.gcode" and "All-Fans_Off-afterPrint_Screensaver_ON.gcode" found on fb group [ELEGOO SATURN 3D Printer Owners](https://www.facebook.com/groups/elegoosaturn/files).

I take a look at the 2 fb files without seeing anything wrong, and I already used your acceleration file with success and without this too long printing time... So I wonder what happened : any idea ?

**Comment by sn4k3** [comment]
> I already used your acceleration file with success and without this too long printing time

Same file and objects?
Double check file with UVtools, look at lifts and speeds. If you somehow print a old file with TSMC or slow feeds thats why

**Reply by Alomphega** [reply]
mmm, I don't think so, because I have already checked, re-checked, and never see something in the UVTools file which could do this... 

Since that issue, I have filled my printer with the original file (BTW thank you for your advice and your "uvtools_dump_eeprom.gcode" file !) 
and re-applied both the "All-Fans_Off-afterPrint_Screensaver_ON.gcode" and your excellent "uvtools_acceleration.gcode" : 
Boom, this works as it should work from now ! 
So I guess the guilty is my aborted try of "Stir.gcode" because I stopped it before it stops by itself, and maybe doing this I caused something wrong in the printer system software.

**Reply by sn4k3** [reply]
Glad that  you find the problem, wierd, but from this "quality" firmwares i don't doubt anything is possible
